notebooks/env.ipynb

```diff
@@ -0,0 +1,605 @@
+      "execution_count": 9,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "env = GridEnv()\n",
+        "env_spec = env.env_spec\n",
+        "obs_spec, act_spec = env_spec.observation_space[0], env_spec.action_space[0]\n",
+        "print(\"Environment name : \\n\", env_spec.env_name)\n",
+        "print(\"Environment observation space: \\n\", obs_spec)\n",
+        "print(\"Environment action space: \\n\", act_spec)\n",
+        "print(\"Environment info: \\n\", env_spec.env_info)"
+      ],
+      "metadata": {
+        "id": "wBNYTJw1xxFD",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "8333bf10-6c97-4ba6-c95f-cf20b5a3aac3"
+      },
+      "execution_count": 10,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Environment name : \n",
+            " GridEnv\n",
+            "Environment observation space: \n",
+            " Discrete(7, start=3)\n",
+            "Environment action space: \n",
+            " Discrete(3, start=-1)\n",
+            "Environment info: \n",
+            " {}\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "terminated = truncated = False\n",
+        "env.reset()\n",
+        "\n",
+        "while not terminated and not truncated:\n",
+        "    obs, reward, terminated, truncated, turn, info = env.step(act_spec.sample())\n",
+        "    print(\"Cell {}, Reward {}\".format(obs, reward))\n",
+        "\n",
+        "env.close()"
+      ],
+      "metadata": {
+        "id": "9zEL32h-yTgx",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "4abfe938-a57c-450d-e774-44f44bc8ac3a"
+      },
+      "execution_count": 12,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Cell 1, Reward 1\n",
+            "Cell 2, Reward 0\n",
+            "Cell 2, Reward 0\n",
+            "Cell 2, Reward 0\n",
+            "Cell 1, Reward 1\n",
+            "Cell 2, Reward 0\n",
+            "Cell 2, Reward 0\n",
+            "Cell 2, Reward 0\n",
+            "Cell 1, Reward 1\n",
+            "Cell 0, Reward 0\n",
+            "Cell 1, Reward 1\n",
+            "Cell 0, Reward 0\n",
+            "Cell 0, Reward 0\n",
+            "Cell -1, Reward 0\n",
+            "Cell -2, Reward 0\n",
+            "Cell -1, Reward 0\n",
+            "Cell -1, Reward 0\n",
+            "Cell 0, Reward 0\n",
+            "Cell 1, Reward 1\n",
+            "Cell 0, Reward 0\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#### Registering environments\n",
+        "The registry module `hive.utils.registry` is used to register classes in the RLHive Registry. Consider registering `GridEnv` we created before:"
```

Comment:
Personally, I would like to know a simple use case for the information given in this subsection. Simply putting out that we can register envs in RLHive, raise the question of _so what?_. 

I think the message could be if you register your custom env, then you can use it directly in the code or sth. 

Do you agree? and if so please make the required changes.